src/routing/index.js

```diff
@@ -56,7 +59,10 @@ const Routing = () => {
         component={ResidentialCourse}
       />
       <Route path={PATHS.COURSE_CONTENT} component={CourseContent} />
```

Comment:
Is this route still used?  I couldn't find a link to it.

Reply:
@jschanker  this is not used anymore but some of the component from it certainly are
its there and will be removed as soon as we refactor everything in code 
@komalahire  and @Poonam-Singh-Bagh are working and knows the basis of it

Reply:
@Poonam-Singh-Bagh  we will need a UI component will you do it??